js/vote.js

```diff
@@ -144,8 +142,18 @@ $(document).ready(function () {
 			commentBox: comment.value.trim()
 		};
 		$('.new-comment .icon-loading-small').show();
-		$.post(form.action, data, function (data) {
-			$('.comments .comment:first').after('<div class="comment"><div class="comment-header"><span class="comment-date">' + data.date + '</span>' + data.userName + '</div><div class="wordwrap comment-content">' + data.comment + '</div></div>');
+		$.post(form.action, data, function(data) {
+		var newCommentElement = '<li class="comment column"> ' +
```

Comment:
We should use [Handlebars ](https://docs.nextcloud.com/server/12/developer_manual/app/tutorial.html?highlight=handlebar#creating-a-navigation) here.

Reply:
Awesome. Was the missing link for me. I will go into it later. 👍